doc/sphinx/Clusters_from_Scratch/ap-configuration.rst

```diff
@@ -226,22 +196,26 @@ Cluster Options
     Cluster Properties:
      cluster-infrastructure: corosync
      cluster-name: mycluster
-     dc-version: 2.1.0-3.el8-7c3f660707
+     dc-version: 2.1.2-4.el9-ada5c3b36e2
      have-watchdog: false
      no-quorum-policy: freeze
      stonith-enabled: true
 
-The output shows state information automatically obtained about the cluster, including:
+The output shows cluster-wide configuration options, as well as some baseline-
+level state information. The output includes:
 
 * **cluster-infrastructure** - the cluster communications layer in use
 * **cluster-name** - the cluster name chosen by the administrator when the cluster was created
 * **dc-version** - the version (including upstream source-code hash) of Pacemaker
   used on the Designated Controller, which is the node elected to determine what
   actions are needed when events occur
+* **have-watchdog** - whether watchdog integration is enabled; set
+  automatically when SBD is enabled
+* **stonith-enabled=true** - whether the cluster is allowed to use STONITH resources
```

Comment:
s/=true//

Reply:
Son of a... I fixed this already. Swore I pushed it. Thanks :)